data/core/step2_rasa_init.md

```diff
@@ -91,7 +90,7 @@
     - utter_ask_ready_to_build
 * deny
     - utter_ask_if_problem
-* technical_question OR enter_data OR out_of_scope
```

Comment:
These should not be removed, this is intentional -- if we give them an installation command and they immediately have a technical question, we assume it is a problem with installation and lead them to the detailed installation page

Reply:
Corrected

Reply:
This is still here in the latest diff